app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/VendorProfileActivity.java

```diff
@@ -0,0 +1,105 @@
+package com.epfl.neighborfood.neighborfoodandroid.ui.activities;
+
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.epfl.neighborfood.neighborfoodandroid.R;
+
+public class VendorProfileActivity extends AppCompatActivity implements View.OnClickListener{
+
+    ImageView subscribeButton;
+    Boolean subscribed = false;
+    ImageView notificationButton;
+    Boolean notifyOn = false;
+    ImageView facebookIcon,instagramIcon, twitterIcon;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_vendor_profile);
+        notificationButton = findViewById(R.id.notificationId);
+        subscribeButton = findViewById(R.id.SubscribeId);
+        facebookIcon = findViewById(R.id.facebookId);
+        instagramIcon = findViewById(R.id.instagramId);
+        twitterIcon = findViewById(R.id.TwitterId);

```

Comment:
there is a lot of hard coding and duplication of code for the social media buttons, Is there a way to put them in a list ?

Reply:
This is just temporary since social media buttons will be dynamically added later